src/css/custom.scss

```diff
@@ -464,3 +464,21 @@ code {
     height: 40px;
     align-items: center;
 }
+
+.datepicker-container {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+}
+
+.datepicker-row {
+    display: flex;
+    flex-direction: row;
+    gap: 1rem;
+    justify-content: space-around;
+}
+
+.datepicker-model {
```

Comment:
If these are intended as reusable styles, forcing the width doesn't seem like a good idea. (On a quick test the `!imporant` doesn't affect things, but I may well be wrong about that.) If they're not intended as reusable, then the class name could be something less generic.

Reply:
Yep those do nothing. Weird though could have sweared that back when I added those, those did something. Anyways removed the importants.